### DISCUSSION TITLE
Set a variable value using Tailwind

### DISCUSSION THREAD

**Discussion by Timonney** [tailwindlabs/tailwindcss#9844] [Help] [answered]
Hi,

I was wondering if it could be a good idea to allow to set a variable value using the Tailwind sytax.

A context in which it can be usefull is to personalize SVG images that are reference like that

```
<svg height="60" width="60" style="--test-color: blue;">
  <use href="icons.svg#my-icon"></use>
</svg>
```

Where '--test-color' is used inside the svg to specify a color.

The idea would be to add a syntax to set the value of '--test-color' using the Tailwind color palette.

Maybe a syntax like that :
`[var(--test-color)]-sky-400`

**Comment by wongjn** [comment, marked as answer]
In case you weren't aware, this is already possible via [arbitrary properties](https://tailwindcss.com/docs/adding-custom-styles#arbitrary-properties): `[--test-color:theme(colors.sky.400)]`.

**Reply by Timonney** [reply]
Thank you, I missed the part where you could set variables with the arbitrary properties
